webrender/src/device/gfx/device.rs

```diff
@@ -1267,15 +1275,18 @@ impl<B: hal::Backend> Device<B> {
         _program_id: &ProgramId,
         projection: &Transform3D<f32, euclid::UnknownUnit, euclid::UnknownUnit>,
     ) {
-        self.bound_locals.uTransform = projection.to_row_arrays();
+        if self.bound_locals.uTransform != projection.to_row_arrays() {
```

Comment:
looks like we are flattening the projection transform twice here, should keep `projection.to_raw_arrays()` around